tests/Zend/Filter/HtmlEntitiesTest.php

```diff
@@ -214,7 +214,7 @@ public function testQuoteStyleQuotesEncodeNone()
      */
     public function testCorrectsForEncodingMismatch()
     {
-        if (version_compare(phpversion(), '5.4', '>=')) {
+        if (PHP_VERSION_ID >= 70000 && PHP_VERSION_ID < 70100) {
             $this->markTestIncomplete('Tested feature ZF-11344 is not available because of PHP bug #63450');
         }
```

Comment:
interesting. Was the mentioned bug fixed in later php 5 versions but got reintroduced in php 7.0?

Reply:
I don't really know. I was probably playing around and saw that the bug only occurred on 7.0